samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Format/cs/starting3.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Text;
+ 
+class Example
+{
+   public static void Main()
+   {
+       // <Snippet33>
+       int[] years = { 2013, 2014, 2015 };
```

Comment:
This sample would be a great example for interpolated strings:

```csharp
int[] years = { 2013, 2014, 2015 };
int[] population = { 1025632, 1105967, 1148203 };
var sb = new StringBuilder();
sb.Append($"{"Year",6} {"Population",15}\n\n");
for (int index = 0; index < years.Length; index++)
     sb.Append($"{years[index],6} {population[index],15:N0}\n");       
Console.WriteLine(sb.ToString());
```

I saw a number of other `{0}` and `{1}` code examples. It might be nice to update those as well.

Reply:
@BillWagner,, the problem with using string interpolation in these examples is that eliminates the call to the String.Format method, which is the method that is being documented. I've added a note that string interpolation is the recommended alternative, and possibly I should add an item to the Q&A section that compares String.Format with string interpolation, but I think that we should leave the calls to String.Format.

Reply:
Good points.